public/locales/fr/goTo.json.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+// For more information, check the common.json.ts file
+
+export default {
+  "search": "Rechercher",
```

Comment:
```suggestion
  "search": "Allez vers...",
```
Ou quelque chose du style ? Par exemple en anglais ça peut donner "Jump to"